source/tutorials/moving-pictures.rst

```diff
@@ -115,10 +118,49 @@ In the ``demux.qzv`` quality plots, we see that the quality of the initial bases
      --i-demultiplexed-seqs demux.qza \
      --p-trim-left 0 \
      --p-trunc-len 120 \
-     --o-representative-sequences rep-seqs.qza \
-     --o-table table.qza
+     --o-representative-sequences rep-seqs-dada2.qza \
+     --o-table table-dada2.qza
+
+If you'd like to continue the tutorial using this FeatureTable (opposed to the Deblur feature table generated in *Option 2*), run the following commands.
+
+.. command-block::
```

Comment:
Comment from @ebolyen: make a note that the file names aren't important, because these `mv` commands might lead the users to think they are.